lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -243,15 +278,18 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
     func numberOfComponents(in pickerView: UIPickerView) -> Int {
         return 1
     }
+}
+
+extension AddComicViewController: UITextFieldDelegate {
 
     // MARK: - UITextFieldDelegate
 
     func textFieldDidBeginEditing(_ textField: UITextField) {
         activeField = textField
-         guard let comicTitleText = comicTitleTextField.text,
-               let bottomBorderSubLayer = textField.layer.sublayers,
-               let comicTitleTextFieldSubLayers = comicTitleTextField.layer.sublayers else {
-            return
+        guard let comicTitleText = comicTitleTextField.text,
+            let bottomBorderSubLayer = textField.layer.sublayers,
+            let comicTitleTextFieldSubLayers = comicTitleTextField.layer.sublayers else {
```

Comment:
`else { return }` should be a single line indented at the same level as the `let`s